### PR TITLE
Capgo integration documentation

### PR DESCRIPTION
**Pull request by icarusdust**

### CODE REVIEW

File: content/integrations/capgo-integration.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+CapacitorUpdater.notifyAppReady()
+```
+4. Install **capacitor-updater** by running the following commands in your **codemagic.yaml**
+```
+    - name: Install npm dependencies for Ionic project
+      script: |
+        npm install @capgo/capacitor-updater
+        npx @capgo/cli login $CAPGO_TOKEN
+```
+**$CAPGO_TOKEN** is your login token and can be found under your Capgo account. Environment variables can be added in the Codemagic web app using the ‘Environment variables’ tab. You can then and import your variable groups into your codemagic.yaml. For example, if you named your variable group ‘browserstack_credentials’, you would import it as follows:
```
Comment (nevercode-kevin) [inline content/integrations/capgo-integration.md:34]:
```suggestion
**$CAPGO_TOKEN** is your login token and can be found under your Capgo account. Environment variables can be added in the Codemagic web app using the Environment variables tab. You can then import your variable groups into your codemagic.yaml. For example, if you named your variable group ‘capgo_credentials’, you would import it as follows:
```

File: content/integrations/capgo-integration.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+```
+workflows:
+  workflow-name:
+    environment:
+      groups:
+        - browserstack_credentials
```
Comment (nevercode-kevin) [inline content/integrations/capgo-integration.md:41]:
```suggestion
        - capgo_credentials
```

File: content/integrations/capgo-integration.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+      script: |
+        npx @capgo/cli add 
+        npx @capgo/cli upload
+```
+When uploading app versions to **Capgo**, executing the following command will submit updates to all user (if production channel is set to public):
+```
+npx @capgo/cli@latest upload -c production
+```
+
+As soon as users start installing app versions on their devices, a device list will be visible in the Capgo UI and then any of them can be pivked in order to let specific groupd of users know about updates shipped with version uploads.
```
Comment (nevercode-kevin) [inline content/integrations/capgo-integration.md:62]:
```suggestion
As soon as users start installing app versions on their devices, a device list will be visible in the Capgo UI and then any of them can be picked in order to let specific groups of users know about updates shipped with version uploads.
```

File: content/integrations/capgo-integration.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
+---
+title: Capgo integration
+description: How to integrate your workflows with Capgo using codemagic.yaml
+weight: 7
+---
+
+## Getting Started
+
+[Capgo](https://capgo.app/) is an instant updater for Capacitor apps and allows you user to try beta version without the need of TestFlight, or Google beta process. It is possible to integrate the flow with **Codemagic**.
```
Comment (nevercode-kevin) [inline content/integrations/capgo-integration.md:9]:
```suggestion
[Capgo](https://capgo.app/) allows you to deploy live updates for Capacitor apps after they have been published to the App Store or Google Play.
```

**Review by nevercode-kevin — CHANGES_REQUESTED** [review]

**Comment by icarusdust** [comment]
Done!

File: content/integrations/capgo-integration.md [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+---
+
+## Getting Started
+
+[Capgo](https://capgo.app/) allows you to deploy live updates for Capacitor apps after they have been published to the App Store or Google Play.
+
+In order to get live updates in your Capgo account via Codemagic, you will need to follow the steps:
+
+1. Sign up with Capgo to get your login token
+2. Add the foolowing part in your **capacitor.config.json**
```
Comment (nevercode-kevin) [inline content/integrations/capgo-integration.md:14]:
```suggestion
In order to get live updates in your Capgo account via Codemagic, you will need to complete the following the steps:

1. Sign up with Capgo to get your login token
2. Add the following part in your **capacitor.config.json**
```

**Review by nevercode-kevin — CHANGES_REQUESTED** [review]

File: content/integrations/capgo-integration.md (hunk 5 above)
Comment (icarusdust) [reply content/integrations/capgo-integration.md]: Done

**Review by nevercode-kevin — APPROVED** [review]
LGTM

**Pull request merged**